zjit/src/profile.rs

```diff
@@ -66,6 +66,8 @@ fn profile_insn(bare_opcode: ruby_vminsn_type, ec: EcPtr) {
         YARVINSN_opt_ge    => profile_operands(profiler, profile, 2),
         YARVINSN_opt_and   => profile_operands(profiler, profile, 2),
         YARVINSN_opt_or    => profile_operands(profiler, profile, 2),
+        YARVINSN_opt_empty_p  => profile_operands(profiler, profile, 1),
```

Comment:
```suggestion
        YARVINSN_opt_empty_p => profile_operands(profiler, profile, 1),
```

seems like an unnecessary space if it's not for aligning indentation.